### ISSUE TITLE
Migrate old core features of AB

### ISSUE THREAD

**Issue by seekheart** [progdisc/awesome_bot#1]
Migrate the following features from old [AB](https://github.com/progdisc/AwesomeBot/tree/master/commands).

- [ ] evaluation of various programming language through repl
- [ ] help command for various topics
- [ ] comic links
- [ ] pro command to help members connect with helpers in a language
- [ ] references storage command to display reference material
- [ ] server command to show server stats
- [ ] stream command for streamers
- [ ] uptime command
- [ ] voting command
- [ ] weather command for checking weather
- [ ] xkcd command

**Comment by Heasummn** [comment]
The stream command, voting command, and help command are rather unnecessary. AB and its commands don't have any general theme or purpose. Before migrating them all over, we should discuss what the role of AB on the server is.

**Comment by seekheart** [comment]
@Heasummn the only features I would like to keep are the eval and pro commands. I think this ticket should be more of a discussion rather than implementation.

**Comment by Heasummn** [comment]
I think we should keep the xkcd and stats commands because those are used and are nice. However, all the other commands are extraneous/rarely used. What other features do you think should be added to AB? We don't really have a specific job for AB, it's just a bot that does a variety of random commands.

**Comment by seekheart** [comment]
I was thinking AB could help facilitate matching newbies with people who can help in lang X and also do some small casual things. main focus of ab should be helping.

**Comment by saethlin** [comment]
Should we move away from repl.it? Their site says they're not accepting any new code evaluation API customers; could this suggest it will disappear in the future?

**Comment by seekheart** [comment]
good idea, what would be a good replacement? Should we strike out stuff we don't want? (can someone edit my post or tell me which ones)

**Comment by saethlin** [comment]
ideone sounds like a good idea, there's a Python API here: https://github.com/jschaf/ideone-api

**Comment by seekheart** [comment]
I like this api, nice find!
